microsoft-edge/devtools-guide-chromium/whats-new/2022/02/devtools.md

```diff
@@ -0,0 +1,154 @@
+---
+title: What's new in DevTools (Microsoft Edge 98)
+description: 2021 year-in-review blog post.  Automate IE mode with Internet Explorer Driver.  Emulate forced colors mode.  Tooltips for Activity icons in the Performance tool Event Log.  Shallow sizes in the Memory tool now shown as decimal values.  Network tool's Search textbox can now be resized.  And more.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/02/2022
+---
+# What's New in DevTools (Microsoft Edge 98)
+
+<!--
+todo:
+compare result against work items
+-->
+
+The following sections list the announcements from the Microsoft Edge DevTools team.  To try the latest features of DevTools and the Microsoft Edge DevTools extension for Visual Studio Code, read these announcements.  To stay up to date with the latest and greatest features in your developer tools, download the [Microsoft Edge preview channels](https://www.microsoftedgeinsider.com/download) and [follow the Microsoft Edge DevTools team on Twitter](https://twitter.com/EdgeDevTools).
```

Comment:
Sentence 1 needs a re-do, given the way the page is structured into a top vs. bottom part, with the latest approach for the final h2 section:
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2022/02/devtools?branch=pr-en-us-1713#announcements-from-the-chromium-project